ocaml/xapi/extauth_plugin_ADwinbind.mli

```diff
@@ -75,3 +75,15 @@ module Migrate_from_pbis : sig
 
   val parse_value_from_pbis : string -> string
 end
+
+module type HostsConf = sig
```

Comment:
Maybe:
```
type t = string list

val join : name:string -> domain:string -> t -> t
```

It would be even stronger if t could be made abstract but I'm not calling to make it complicated for the sake of it.

Reply:
yeah, it can be. But I am not going to do it to keep simplify.